setup/kubernetes/google.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Google Kubernetes Engine 
+description: Learn how to set up a GKE cluster for your Coder deployment.
+---
+
+This guide shows you how to set up a Google Kubernetes Engine (GKE) cluster to
+which Coder can deploy.
+
+## Prerequisites
+
+Before proceeding, make sure that the [gcloud
+CLI](https://cloud.google.com/sdk/docs/quickstarts) is installed on your machine
+and configured to interact with your Google Cloud Platform account.
+
+## Set Up the GKE Cluster
+
+The following will spin up a Kubernetes cluster using the `gcloud` command (be
+sure to replace the parameters (specifically `PROJECT_ID` and
+`NEW_CLUSTER_NAME`) as needed to reflect the needs of your environment).
+
+```bash
+gcloud beta container --project "PROJECT_ID" \
+clusters create "NEW_CLUSTER_NAME"
```

Comment:
```suggestion
clusters create "NEW_CLUSTER_NAME" \
```

This just needs a backslash at the end since the lines below are continuations of the same command